src/test/lib/CMakeLists.txt

```diff
@@ -21,7 +21,9 @@ set(SOURCES
 
 add_library(testlib STATIC ${SOURCES})
 
-target_include_directories(testlib PRIVATE ${PROJECT_SOURCE_DIR}/src/test)
+target_include_directories(testlib
+    PRIVATE ${PROJECT_SOURCE_DIR}/src/test
+    PRIVATE ${PROJECT_SOURCE_DIR}/src/core)
```

Comment:
I wonder if it's worth moving `quicdef.h` to `.\inc\quic_defines.h`.

Reply:
hmm, it is difficult for me to decide. it should be different PR if needed